taipy/core/data/parquet.py

```diff
@@ -189,7 +189,7 @@ def _read_from_path(self, path: Optional[str] = None, **read_kwargs) -> Any:
 
         # return None if data was never written
         if not self.last_edit_date:
-            self._DataNode__logger.warning(
+            self._DataNode.__logger.warning(
```

Comment:
I think this change is wrong

Reply:
yeah it is, but linter is complaining that it doesn't recognize _DataNode__logger :/

Reply:
Yes, because it doesn't exist. But `self._DataNode__logger` does. Can you revert this change?

Reply:
why don't we set the logger as a hidden variable in stead of private? That would be cleaner for me. No?

Reply:
Yes, I think we missed this change when reviewed the work from a contributor.